sql/core/src/main/scala/org/apache/spark/sql/DataFrameNaFunctions.scala

```diff
@@ -130,20 +130,20 @@ final class DataFrameNaFunctions private[sql](df: DataFrame) {
    *
```

Comment:
@cloud-fan what's the behavior of "*" for `fill()`?

```
scala> right.show
+----+----+
|col1|col2|
+----+----+
|   1|   2|
|   3|null|
+----+----+


scala> right.na.fill("hello", Seq("*")).show
+----+----+
|col1|col2|
+----+----+
|   1|   2|
|   3|null|
+----+----+
```
Should `*` work for `fill` such that all columns are affected?

Reply:
Actually, `drop` shows different behavior for `*` since it's using `df.resolve` instead of `df.col`:
```Scala
scala> right.na.drop(Seq("*")).show
org.apache.spark.sql.AnalysisException: Cannot resolve column name "*" among (col1, col2);
  at org.apache.spark.sql.Dataset$$anonfun$resolve$1.apply(Dataset.scala:223)

```

Reply:
good question. Personally I think we don't need to resolve `*` here, but to be safe let's keep the previous behavior unchanged.